qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: 1.1. Sign in to App with Google account
+
+Scenario: Sign in with Google account granted
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	When user hits "Sign in with Google" button
+	And "Google account authentication" is granted by Google
+	Then user is signed in to the app
```

Comment:
each then step provides expected outcome and related to the previous given and when steps.